_layouts/page.html

```diff
@@ -6,7 +6,25 @@ <h1>{{page.chapter}}. {{page.title}}</h1>
 
 {% assign full_name = page.chapter | prepend: "chapter" %}
 {% assign post_list = site.categories[full_name] | sort_natural %}
+
+<!-- Get first post and show it -->
+{% assign first_post = post_list | first %}
+{{first_post.content}}
+
+<!-- Remove first element from post_list which is already shown above. -->
+{% assign post_list = post_list | shift %}  
+
+<!-- List up the posts in the chapter -->
+<table>
 {% for post in post_list %}
-  <h2><a href="{{ post.url }}">{{ post.title }}</a></h2>
-  {{post.content}}
-{% endfor %}
\ No newline at end of file
+  {% assign title_parts = post.title | split: " " %}
+  {% assign subchapter = title_parts | first %}
+  {% assign title = post.title | remove: subchapter %}
+  <tr>
+    <th style="text-align:left; vertical-align: middle;" >{{subchapter}}</th>
+    <th style="text-align:left; vertical-align: middle;" >
```

Comment:
```suggestion
    <th style="text-align:middle; vertical-align: middle;" >{{subchapter}}</th>
    <th style="color: blue; text-align:middle; vertical-align: middle;" >
```
![image](https://user-images.githubusercontent.com/36983960/112709560-02846300-8efe-11eb-8e6a-ed9763eb1fb6.png)
* 개인적으로 오버뷰뷰는 테이블 형태가 보기 좋은 것 같습니다. 첨부 그림 내 빨간줄처럼 가운데 정렬되고 링크가 파란색(?) 으로 보이면 더 링크라는게 잘 보일 것 같습니다.
  * 바로 적용해보려고 했는데요, 위 첨부한것은 마크다운 테이블입니다. html 언어에서 링크에 색깔을 먹여도 적용되지가 않네요..ㅠㅠ text-align 만 middle로 수정해봤습니다. 혹시 링크에도 색깔을 넣는 방법 아실까요?
* 민주님이 제안해주신 모양도 예쁜데, related post에 있으면 더 좋을것 같다는 의견입니다.
* 다만 이제 첫화면에서 2장 페이지 클릭하면 오버뷰가 보이고, 세세 내용은 클릭해서 들어가야 하는데요. 오버뷰 아래에서 2장 전체 내용이 쭉 이어지는게(기존처럼) 사용자 입장에서 읽기 더 편할 것 같습니다.

Reply:
> > sub-chapter number의 길이가 모두 달라 Table이 아니면 예쁘게 만들기 어려웠습니다
> 
> 길이가 다른게 문제가 되나요? 저는 이정도도 충분하다고 생각했습니다 :)
> <img alt="스크린샷 2021-03-27 오후 12 41 13" width="1920" src="https://user-images.githubusercontent.com/42792260/112709099-c222e600-8ef9-11eb-86ce-e16689bf63b4.png">

처음에 올려주신 방식대로 했는데, Title이 시작하는 지점이 sub chapter의 depth에 따라 각기 다르게 시작해서 가독성이 떨어진다고 생각했습니다ㅠ

![image](https://user-images.githubusercontent.com/31348169/112717854-88ba9c80-8f32-11eb-9cbb-f16410dbbf50.png)

Reply:
- 원래 파란 줄로 설정되어 있었는데, 본 PR에서 hyperlink의 색깔을 빼도록 했습니다. 파란 색으로 원복하는 것이 나을까요?
-  개인적으로는 기존처럼 한 페이지에서 모든 포스팅을 확인할 수 있도록 하면 한 페이지에 너무 많은 정보가 담기게 되어 불편한 것 같습니다.

Reply:
아하.. 댓글문제도 생기는군요. 책의 한 장이 쭈르륵 보이는건 무척 좋았는데. 비슷하게 지원할 수 있는 방법을 찾으면 좋을 것 같습니다 ㅎㅎ

Reply:
> 기존처럼 하는 경우 댓글을 작성할 수 없다는 문제도 있었습니다ㅠ

댓글을 한 포스팅기준으로 달 수 없다는 의미이신가요? (질문)

Reply:
넵넵 여러 개가 한 번에 보이는 경우에는 댓글 기능이 지원되지 않습니다. 추가해도 되지만 정보량이 너무 많아 댓글 기능을 넣는 것은 좋지 않을 것 같습니다. 별도 기능으로 개별 chapter의 모든 포스팅이 보이도록 하는 것이 좋을 것 같습니다.